renderdoc/driver/gl/gl_renderstate.cpp

```diff
@@ -637,6 +637,7 @@ bool GLRenderState::CheckEnableDisableParam(GLenum pname)
       case eGL_PROGRAM_POINT_SIZE:
       case eGL_PRIMITIVE_RESTART:
       case eGL_TEXTURE_CUBE_MAP_SEAMLESS:
+      case eGL_FRAMEBUFFER_SRGB:
```

Comment:
I know there's some weird behaviour for GLES around SRGB where it varies subtly from GL. I don't remember the details but this came up a while ago when I was talking about GLES.

From what I can see GL_FRAMEBUFFER_SRGB isn't supported on GLES but instead of acting like it's always disabled (which is what these unsupported enabled bits end up being) it acts like it's always _enabled_. What is the impact of leaving the enabled bit false? Is there any fixup needed elsewhere in the codebase to handle the different meaning for GLES/GL, or is it better to have it just to silently default to enabled on GLES?